src/Handler.php

```diff
@@ -42,12 +42,12 @@ public static function setPermissions(Event $event)
             }
 
             try {
-                if (chmod($dir, $permission)) {
+                if (chmod($dir, octdec($permission))) {
```

Comment:
For simplicity, I think fine to keep it as it is.

Reply:
It sets the wrong permissions as it is. I'm using php8.